### DISCUSSION TITLE
Seeking Developers with Published Alexa Skills to Test Our New Tool ($20 Reward)

### DISCUSSION THREAD

**Discussion by Yanjingwen111** [community/community#110895] [Discussions]
Hello developers! 

Our lab has been working hard on a new tool designed to make life easier for developers, especially when it comes to privacy. We're excited to introduce this tool to you and are currently in the testing phase to ensure its usability. 

What we need: 
We're looking for developers who have published Alexa skills (using JavaScript as back-end program language) to test our new tool. The primary focus is to assess its usability in generating privacy notices. 

Your role: Test the tool to generate a privacy notice for your published skills. The testing is expected to take less than 10 minutes. Provide feedback on the tool's usability, effectiveness, and any bugs or improvements you might suggest.

Reward: We value your time and effort, so we're offering a $20 reward for developers who complete the testing and provide constructive feedback. 

How to participate: If you're interested in helping us test this tool and earn a reward in the process, please comment below or send me a DM or send email to privacynoticelab@gmail.com. We'll provide you with access to the tool and further instructions. 

Thank you for considering this opportunity, and we look forward to your participation!

**Comment by queenofcorgis** [comment]
Hi @Yanjingwen111 , we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
